portfolio/src/main/webapp/kite_gallery.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html id = "gallery">
+  <head>
+    <meta charset="UTF-8">
+    <title>Kite Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="gallery_content">
+      <h1 id = "g_title">My Kite Image Gallery</h1>
```

Comment:
minor comment: there are a few more cases of whitespaces around HTML attributes.